src/App.vue

```diff
@@ -412,6 +416,22 @@ export default {
     this.loadFromLocalStorage();
   },
   methods: {
+    // eslint-disable-next-line func-names
```

Comment:
if you were getting an eslint-error, then leave this line as it is. But, if not, you were getting an eslint warning which you can remove this line. Same goes to line 427

Reply:
I agree with the previous observation, it is much better (or the right way) to "debounce" this method in its definition rather that changing it during the component creation. About the `eslint-error` maybe that could be changed in a different ticket.